environments/Map2DEnvironment.cpp

```diff
@@ -352,29 +352,29 @@ bool MapEnvironment::GetNext8Successor(const xyLoc &currOpenNode, const xyLoc &g
 //	return false;
 }
 
-void MapEnvironment::GetActions(const xyLoc &loc, std::vector<tDirection> &actions) const
+void MapEnvironment::GetActions(const xyLoc &loc, std::vector<tDirection> &actions) const // can agent go to diff loc?
 {
 	bool up=false, down=false;
 	if ((map->CanStep(loc.x, loc.y, loc.x, loc.y+1)))
 	{
 		down = true;
-		actions.push_back(kS);
+		actions.push_back(kS); // South
 	}
 	if ((map->CanStep(loc.x, loc.y, loc.x, loc.y-1)))
 	{
 		up = true;
-		actions.push_back(kN);
+		actions.push_back(kN); // North added to actions
```

Comment:
These comments are your own notes and should't be checked in. Can you make a new pull request without the changes to Map2DEnvironment included?